packages/generator-rest/test/rest-generator.spec.ts

```diff
@@ -5,7 +5,7 @@ import * as fs from 'fs-extra';
 import { generateRest } from '../src/generator';
 
 describe('rest generator test', () => {
-  const inputDir = path.resolve(__dirname, 'resources');
+  const inputDir = path.resolve(__dirname, 'resources', 'test-apis');
```

Comment:
I understand that the integration test needs the token, but you could add a test on the generated classes and methods.

Reply:
So that the generated class has the expected files and the class has the method, but this is also implicitliy tested already